### PR TITLE
Js sdk

### PR DESCRIPTION
**Pull request by AmnaRiaz**
Read and write function's sdk

### CODE REVIEW

**Comment by Zaryab-Programmer** [comment]
checking the code

File: src/contractcall.js [hunk 1]
```diff
@@ -1,83 +1,182 @@
...
+            }
+        });
+    });
+
+}
+
+async function Update() {
+
+    //console.log("abi", abi.abi);
+    let data = contract.methods.update().encodeABI();
```
Comment (Zaryab-Programmer) [inline src/contractcall.js:107]: contract update function has the parameter, but there is no parameter given

File: src/contractcall.js [hunk 2]
```diff
@@ -1,83 +1,182 @@
...
+        });
+    });
+
+}
+
+
+async function Burn() {
+
+    //console.log("abi", abi.abi);
+    let data = contract.methods.burnNfid().encodeABI();
```
Comment (Zaryab-Programmer) [inline src/contractcall.js:143]: burnNfid fucntion has also the parameter but there is no parameter given to it

**Review by Zaryab-Programmer — COMMENTED** [review]
these are the review of changes

File: src/contractcall.js (hunk 1 above)
Comment (AmnaRiaz) [reply src/contractcall.js:107]: khudi ni dye parameters jisny krna ho wo  kr k dekh ly 😎

**Pull request merged**